core/execution/market.go

```diff
@@ -3166,16 +3207,34 @@ func (m *Market) commandLiquidityAuction(ctx context.Context) {
 func (m *Market) tradingTerminated(ctx context.Context, tt bool) {
 	m.mu.Lock()
 	defer m.mu.Unlock()
-	m.mkt.State = types.MarketStateTradingTerminated
-	m.mkt.TradingMode = types.MarketTradingModeNoTrading
-	m.broker.Send(events.NewMarketUpdatedEvent(ctx, *m.mkt))
-	m.stateChanged = true
 
-	if m.settlementPriceInMarket == nil {
-		m.log.Debug("no settlement price", logging.MarketID(m.GetID()))
+	if m.mkt.State != types.MarketStateProposed && m.mkt.State != types.MarketStatePending {
```

Comment:
shouldn't this only be != MarketStateProposed? why pending too?

Reply:
Pending it he OpeningAuction

Reply:
I know but I'm saying that this should be legit to terminate in an opening auction, that's not the same as terminate a market pre enactment, no?

Reply:
I think it is the same.

Reply:
as confirmed in spec, this is fine - proposed and pending should be cancelled if terminated.